test/Interop/Cxx/class/type-classification-non-trivial-irgen.swift

```diff
@@ -0,0 +1,94 @@
+// RUN: %target-swift-frontend -enable-cxx-interop -I %S/Inputs %s -emit-ir | %FileCheck %s
+
+import TypeClassification
+
+// TODO: C++ objects with destructors should be tested here once we fully
```

Comment:
This is unfortunate (but also unrelated to this change). I was meaning to submit a fix for this and forgot. Maybe I'll have time this weekend. Otherwise, I'll do it next week.

Reply:
See #32591.